org/allPRs.ts

```diff
@@ -215,7 +215,21 @@ export const embeddedFilesURLMismatch = async() => {
     await privacyConfigMismatch(repo, modifiedFiles)
 }
 
+export const releaseBranchBSKChangeWarning = async () => {  
+    const branchName = danger.github.pr.head.ref;
+    if (!branchName.startsWith('release/')) return;
```

Comment:
I think we should include `hotfix/` here too, wdyt?

Reply:
Really good point, yep they should apply too. Let me change this right now.